services/archive.py

```diff
@@ -39,27 +38,16 @@ def get_path(self, **kwaargs) -> str:
 # Service class for performing archive operations. Meant to work against the
 # underlying StorageService
 class ArchiveService(object):
+    root = None
     """
     The root level of the archive. In s3 terms,
     this would be the name of the bucket
     """
 
-    root = None
-
-    """
-    Region where the storage is located.
-    """
-    region = None
-
-    """
-    A hash key of the repo for internal storage
-    """
     storage_hash = None
-
     """
-    Boolean. True if enterprise, False if not.
+    A hash key of the repo for internal storage
```

Comment:
is this comment in the wrong place?

Reply:
I guess its the weird way that python doc comments work, as they appear (inside, or) *after* the item they are documenting